accord-core/src/main/java/accord/local/Bootstrap.java

```diff
@@ -136,19 +142,20 @@ void start(SafeCommandStore safeStore0)
                // TODO (required, correcness) : PreLoadContext only works with Seekables, which doesn't allow mixing Keys and Ranges... But Deps has both Keys AND Ranges!
                // TODO (required): is localSyncId even being used anymore
                // ATM all known implementations store ranges in-memory, but this will not be true soon, so this will need to be addressed
-               .flatMap(syncPoint -> node.withEpoch(epoch, () -> store.submit(contextFor(localSyncId, syncPoint.waitFor.keyDeps.keys(), KeyHistory.COMMANDS), safeStore1 -> {
+               .flatMap(syncPoint -> store.submit(empty(), safeStore1 -> {
+                   //TODO (deterministic reply): this changes due to errors and invalidate, so how do we handle this on reply?
```

Comment:
example, we see `valid=[1, 2, 3]`, we schedule `CreateBootstrapCompleteMarkerTransaction` and *before* that runs we update `valid=[1]`... we *store* in journal `valid=[1]` so maybe its fine?